casanovo/casanovo.py

```
@@ -294,13 +304,24 @@ def train(
     verbosity: str,
     force_overwrite: bool,
+    load_all_states: bool,
 ) -> None:
     """Train a Casanovo model on your own data.
 
     TRAIN_PEAK_PATH must be one or more annoated MGF files, such as
     those provided by MassIVE-KB, from which to train a new Casnovo
     model.
     """
+    if model is None and load_all_states:
+        warnings.warn(
```

Ruff (0.15.1)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)